SPECS/containerd2/containerd2.spec

```diff
@@ -5,7 +5,7 @@
 Summary: Industry-standard container runtime
 Name: %{upstream_name}2
 Version: 2.0.0
-Release: 5%{?dist}
+Release: 6%{?dist}
```

Comment:
@dallasd1 Now that `containerd2` provides and obsoletes `containerd`, should we look at removing `containerd` spec altogether to avoid confusion?

Reply:
Yes, we should. @liunan-ms has this in mind and is deciding on whether to maintain a containerd or a containerd2 package, but ultimately retiring one of them